tools/CouchDB_Import_Instruments.php

```diff
@@ -78,83 +109,121 @@ function UpdateDataDicts($Instruments)
 
             $this->CouchDB->replaceDoc(
                 "DataDictionary:$instrument",
-                array(
-                 'Meta'           => array('DataDict' => true),
-                 'DataDictionary' => array($instrument => $Dict),
-                )
+                [
+                    'Meta'           => ['DataDict' => true],
+                    'DataDictionary' => [$instrument => $Dict],
+                ]
             );
         }
     }
 
+    /**
+     * Generates SQL statement
+     *
+     * @param string $tablename Name of the database table
+     *
+     * @return string
+     */
     function generateDocumentSQL(string $tablename) : string
     {
-        $select = "SELECT 
-                        c.PSCID, 
-                        s.Visit_label, 
-                        f.Administration, 
-                        f.Data_entry, 
-                        f.Validity,
-                        f.CommentID, 
-                        CASE WHEN EXISTS (SELECT 'x' FROM conflicts_unresolved cu WHERE f.CommentID=cu.CommentId1 OR f.CommentID=cu.CommentId2) THEN 'Y' ELSE 'N' END AS Conflicts_Exist, 
-                        CASE ddef.Data_entry='Complete' WHEN 1 THEN 'Y' WHEN NULL THEN 'Y' ELSE 'N' END AS DDE_Complete ";
-        $from = "FROM 
-                        flag f 
-                        JOIN session s ON (s.ID=f.SessionID) 
-                        JOIN candidate c ON (c.CandID=s.CandID) 
-                        LEFT JOIN flag ddef ON (ddef.CommentID=CONCAT('DDE_', f.CommentID)) ";
-        $where = "WHERE 
-                        f.CommentID NOT LIKE 'DDE%' 
-                        AND f.Test_name=:inst
-                        AND s.Active='Y' AND c.Active='Y'";
+        $select = <<<'QUERY'
```

Comment:
This is a step backwards. Do not here heredocs to format SQL statements. PHP string literals allow newlines, just add line breaks.